Dockerfile

```diff
@@ -1,14 +1,11 @@
-FROM apache/airflow:2.6.0-python3.10
-
-ENV PYTHONBUFFERED=0 
-ENV AIRFLOW_UID=501
+FROM apache/airflow:2.6.0-python3.8
```

Comment:
why are we using python `3.8`?

Reply:
Because I aimed to keep the same Python version, as in previous tests, to be sure that it is not the version's fault. 
I can change it to 3.10, since it is obvious not the version fault